dask/dataframe/core.py

```diff
@@ -2173,11 +2173,64 @@ def std(
     ):
         axis = self._validate_axis(axis)
         _raise_if_object_series(self, "std")
+
         meta = self._meta_nonempty.std(axis=axis, skipna=skipna)
+        is_df_like = is_dataframe_like(self._meta_nonempty)
+        needs_time_conversion = False
+        numeric_dd = self
+
+        if PANDAS_GT_120:
+            from .io import from_pandas
+            from .numeric import to_numeric
+
+            def convert_to_numeric(s):
```

Comment:
Unfortunately, it seems that `view` has the same numeric value for `NaT`, and skipping nans is a feature of `TimeDeltaArray`.

Reply:
The performance improvement would at least be nice but I agree in that it doesn't solve the issue that would have been helpful here. I am fine with waiting until that PR is merged and refactoring after that to use view, since it won't be much of a change!